metrics/avgload_collector.py

```diff
@@ -43,8 +43,9 @@ def save_metrics():
     data_row = [one_min_out, five_min_out, fifteen_min_out]
     with open(avgload_file, 'a') as metric_csv:
         writer = csv.writer(metric_csv, delimiter=',')
-        datetimeNow = datetime.now()
-        data_row.append(((time.mktime(datetimeNow.timetuple())+ datetimeNow.microsecond) * 1000))
+        datetimeNow = "%.9f" % time.time()
+        datetimeNow = datetimeNow.replace('.', '')
```

Comment:
could you also add a `.replace(',', '')`? I think that depending on the system the Metrics is running in, the float number may have a comma instead of a dot. Could be like these:
```suggestion
        datetimeNow = datetimeNow.replace('.', '').replace(',', '')
```